environment-setup.md

```diff
@@ -1,22 +0,0 @@
-1. install anaconda, install docker
```

Comment:
Can you remove the reference to this doc in the `development-guide.md` file?

Reply:
done